src/Components/Web.JS/src/Rendering/Events/EventDelegator.ts

```diff
@@ -136,6 +136,13 @@ export class EventDelegator {
       return;
     }
 
+    if (!isRendererAttached(this.browserRendererId)) {
+      // when connection closed, it will detachWebRendererInterop, so we need to check if the renderer is still attached
+      const targetElement = evt.target as Element;
+      console.warn(`${evt.type} on ${targetElement.localName}${targetElement.className ? ' with class ' + targetElement.className : ''} is not handled because the renderer is not attached`);
```

Comment:
```suggestion
```
I don't think we need the warn here. We don't log to the console directly, in any case we would use the logger, but more importantly, this is a valid scenario that shouldn't result in a warning.